extensions/interactions/Continue/directives/continue-response.component.html

```diff
@@ -0,0 +1 @@
+{{escapedAnswer}}
```

Comment:
is it working fine for the downgraded component i.e, for AngularJS? (We have a different pattern for angularJS)

Reply:
A downgraded component is just a wrapper around angular components for angularjs, this part is still controlled by angular.